src/shared/scrapers/TW/index.js

```diff
@@ -0,0 +1,114 @@
+import assert from 'assert';
+import * as fetch from '../../lib/fetch/index.js';
+import * as parse from '../../lib/parse.js';
+import datetime from '../../lib/datetime/index.js';
+
+const scraper = {
+  country: 'iso1:TW',
+  level: 'country',
+  aggregate: 'country',
+  url: 'https://covid19dashboard.cdc.gov.tw/dash3',
+  timeseries: false,
+  sources: [
+    {
+      name: 'Taiwan CDC',
+      url: 'https://sites.google.com/cdc.gov.tw/2019-ncov/taiwan',
+      description: ''
+    }
+  ],
+  certValidation: false,
+  type: 'json',
+  scraper: {
+    '0': async function() {
+      const date = process.env.SCRAPE_DATE || datetime.getYYYYMMDD();
+
+      const country = {
+        date,
+        country: this.country,
+        cases: 0,
+        recovered: 0,
+        deaths: 0,
+        tested: 0,
+        active: 0,
+        negative: 0
+      };
```

Comment:
Ideal to not init these. Better for them to be undefined if we can't grab them.
Also no need to get active and negative, they're implied by the other data (cases - recovered = active, tested - cases = negative)

Reply:
Good call @camjc .

Reply:
Yeah, I init'd them for easy incrementing before we used the heard to get dash3. I'll remove them now.

FWIW, `active` absolutely did my head in. I went through everything in my code and then the whole repo. before realising it's automatically-calculated. 🤦

Reply:
Just making the last changes and I'll commit in some minutes. :smiley:

Reply:
Thanks @appastair, it looks great. :-)